packages/compass-e2e-tests/helpers/commands/collection-workspaces.ts

```diff
@@ -56,10 +56,10 @@ export async function navigateWithinCurrentCollectionTabs(
     | 'Indexes'
     | 'Validation' = 'Documents'
 ): Promise<void> {
-  const tab = browser.$(Selectors.collectionSubTab(tabName));
-  const selectedTab = browser.$(Selectors.collectionSubTab(tabName, true));
+  const tab = Selectors.collectionSubTab(tabName);
```

Comment:
We have a lot of cases of this bug. in this case `tab` is a selector (ie. a string) and browser.clickVisible(selector) takes a selector, not an element. And in quite a few places we mistakenly sent it either an element or a promise that will resolve to an element. I'm wondering if we can't just make browser.clickVisible() take either a selector OR an element. OR better yet maybe we can have element custom commands and not just browser ones.